src/OpenTelemetry.Api/.publicApi/net461/PublicAPI.Shipped.txt

```diff
@@ -178,9 +178,7 @@ static OpenTelemetry.Context.Propagation.Propagators.DefaultTextMapPropagator.ge
 static OpenTelemetry.Context.RuntimeContext.ContextSlotType.get -> System.Type
 static OpenTelemetry.Context.RuntimeContext.ContextSlotType.set -> void
 static OpenTelemetry.Context.RuntimeContext.GetSlot<T>(string slotName) -> OpenTelemetry.Context.RuntimeContextSlot<T>
-static OpenTelemetry.Context.RuntimeContext.GetValue<T>(string name) -> T
```

Comment:
The `RuntimeContext` API has mixed usage of "name" & "slotName". I made them all "slotName" to be consistent. I don't think parameter rename is breaking?

Reply:
I thought of a situation where it is probably breaking... name arguments. Like `RuntimeContext.GetValue<bool>(name: "mySlot")`. Probably super rare anyone is a) using this API and b) using named arguments (given there is only one and no defaulted positions) but all the same @reyang you want me to revert?

Reply:
Sorry for the delay here, is this urgent? I will definitely have time to go through the PR this Friday, let me know if you want me to treat it as a priority so I can pop it on top of my list.

Reply:
Not at all! Take your time.

Reply:
I'm fine with the rename 👍 

It is passing the [API Compatiblity CI](https://github.com/open-telemetry/opentelemetry-dotnet/actions/runs/1211315387) which means it's not considered as a breaking change from the tooling perspective.

Like you said, folks who used "named arguments" will get compile time error and they will need to update the code; folks who perform binplace won't notice any break. So I think the "extra burden" is well controlled within the developers' inner loop.